lua/alex/lazy/git-conflict.lua

```
@@ -0,0 +1,7 @@
+return {
+  { 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)